packages/core-app-api/report.api.md

```diff
@@ -637,6 +662,9 @@ export class SamlAuth
   signOut(): Promise<void>;
 }
 
+// @public
+export function showLoginPopup(options: LoginPopupOptions): Promise<any>;
```

Comment:
Thinking `openLoginPopup` might be a better name for this one?

It needs a more explicit return type as well, can't have `any`. Needs to either be narrowed down or switched to `unknown`